packages/console/src/pages/SignInExperience/components/Preview.module.scss

```diff
@@ -4,8 +4,8 @@
   width: 480px;
   padding: 0;
   overflow: hidden;
-  box-shadow: var(--shadow-1);
-  border: 1px solid var(--color-border);
```

Comment:
nit: just double it is intent to be removed.

Reply:
YES, double checked.